### PR TITLE
fix: fix dynamic.d.ts

### PR DESCRIPTION
**Pull request by zyl1314**
##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

update dynamic.d.ts

### CODE REVIEW

File: packages/dva/dynamic.d.ts [hunk 1]
```diff
@@ -1,2 +1,11 @@
-declare const dynamic: (resolve: (value?: PromiseLike<any>) => void) => void;
+import { DvaInstance } from 'dva';
+import React from 'react';
+
+interface Config {
+  app: DvaInstance,
+  models?: () => PromiseLike<any>[]
+  component: () => PromiseLike<any>
+}
```
Comment (justjavac) [inline packages/dva/dynamic.d.ts]: also `resolve?` and `LoadingComponent?` fields?

https://github.com/dvajs/dva/blob/master/packages/dva/src/dynamic.js#L60
https://github.com/dvajs/dva/blob/master/packages/dva/src/dynamic.js#L20

Comment (zyl1314) [reply packages/dva/dynamic.d.ts]: already added

**Comment by gitim** [comment]
any chance to merge this?

**Comment by mllongze** [comment]
官方合并了嘛？

**Pull request closed without merging**